towhee/functional/mixins/display.py

```diff
@@ -12,13 +12,120 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import numpy
+import pandas as pd
+from typing import Tuple
+
 from towhee._types import Image
+from towhee.hparam import param_scope
+from towhee.utils import ipython_utils
+
+
+def get_data_on_columns(self, index: Tuple[str]):
+    columns = [[] for _ in index]
+    for x in self:
+        for i in range(len(index)):
+            columns[i].append(getattr(x, index[i]))
+
+    data = {}
+    for i in range(len(index)):
+        data[index[i]] = columns[i]
+    return data
```

Comment:
We could use map() and an inner function here I think

Reply:
Maybe it can be refactored with `to_df` later, I think `to_df` will support passing parameters `columns`, right?

Reply:
Done